### PR TITLE
Patch note #16

### PR DESCRIPTION
**Pull request by clochix**

### CODE REVIEW

File: src/documents/en/hack/releases/patch-notes-16.html.jade [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+    h3 Printit (0.1.10)
+    ul
+        li Fix bad file from stack
+            In `getFileAndLine`, remove the 4 first lines:
+                Error
+                  at Logger.getFileAndLine (.../node_modules/printit/main.js:51:18)
+                  at Logger.format (.../node_modules/printit/main.js:65:26)
+                  at Logger.debug (.../node_modules/printit/main.js:120:32)
+            Fix <a href="https://github.com/cozy/Printit/issues/11" target="_blank">#11</a>
```
Comment (nono) [inline src/documents/en/hack/releases/patch-notes-16.html.jade]: @clochix maybe, we don't need that much details here

**Comment by clochix** [comment]
You're right, I removed the details.

**Comment by clochix** [comment]
I removed a class on Home that made a link look like a button, because the button was broken on small screens.

**Comment by frankrousseau** [comment]
Ok. Thank you @clochix

**Pull request merged**